modules/core/src/main/kotlin/com/metalbear/mirrord/MirrordExecManager.kt

```diff
@@ -198,6 +198,95 @@ class MirrordExecManager(private val service: MirrordProjectService) {
         return executionInfo
     }
 
+    private fun containerStart(
```

Comment:
The code here is almost completely duplicated. Is there a reason not to share code between `start` and `containerStart` to avoid this repetition?